components/director/internal/destinationfetchersvc/client.go

```diff
@@ -183,14 +184,15 @@ func NewClient(instanceConfig config.InstanceConfig, apiConfig DestinationServic
 
 // FetchTenantDestinationsPage returns a page of destinations
 func (c *Client) FetchTenantDestinationsPage(ctx context.Context, page string) (*DestinationResponse, error) {
-	url := c.apiURL + c.apiConfig.EndpointGetTenantDestinations
-	req, err := c.buildRequest(ctx, url, page)
+	fetchURL := c.apiURL + c.apiConfig.EndpointGetTenantDestinations
+	req, err := c.buildFetchRequest(ctx, fetchURL, page)
 	if err != nil {
 		return nil, err
 	}
 
-	log.C(ctx).Infof("Getting destinations page: %s data from: %s\n", page, url)
+	log.C(ctx).Debugf("Getting destinations page: %s data from: %s", page, fetchURL)
```

Comment:
Duplicates the log from line 233 and doesn't bring any useful insights - we could remove it.